### ISSUE TITLE
Some packages mismatch the datasheets

### ISSUE THREAD

**Issue by ONLYA** [DawidCislo/Alternate-KiCad-Library#7]
I found that the provided components like "VOM617A-4" mismatch the package and foot print recommendation described in its datasheet. I have not checked the other symbols yet but I presume that there might be more symbols having the mismatch footprints.

**Comment by DawidCislo** [comment]
Thank you for reporting, please report any further mismatches here. I will fix it once I am actually able to.

**Comment by DawidCislo** [comment]
For now please manually replace the affected footprint fields with:

- VOM617A to VOM617A-9, replace footprint with: Package_SO_AKL:SO-4_4.4x3.9mm_P2.54mm
- VOM618A to VOM618A-9, replace footprint with: Package_SO_AKL:SO-4_4.4x3.9mm_P2.54mm

**Comment by DawidCislo** [comment]
Just a note on possible pad size differences between AKL and manufacturer recommendations:
The recommended footprint size for the same package can vary widely for different manufacturers, and even different assembly houses will have differences in their recommendations. This comes down to many factors including but not limited to:
- Type of solder (leaded vs unleaded, flux type, granularity of the solder balls in the paste)
- Pick and place machine precision (bigger pads are more tolerant to inaccurate placement)
- Coplanarity of the PCB (how flat it is out of the box, how much does it bend)
Solder paste aperture (F.Paste layer) geometry is also critical to ensure a reliable and repeatable joint in any reflow soldering production process.
Most large engineering companies create their own database of known-good footprints tailored to their preferred manufacturing processes. 
This library will never be able to achieve that, and included footprints are only good for hobby projects or prototypes that are typically soldered by hand, not on a whole assembly line. You can still use the AKL as a base and modify the pad and paste aperture geometry for footprints you want to use for a high-volume product, according to the assembly house recommendations and past experiences.
TLDR: Pad size in AKL is designed to accomodate manual soldering and not much else, you will need to adjust most footprints if you plan on mass-producing your design.

**Comment by DawidCislo** [comment]
Release candidate for V4 is out, for now only on GitHub. Your issues have been fixed.
